RUN_ON_SERVER.md

```diff
@@ -10,7 +10,7 @@ sudo apt update && sudo apt upgrade -y && sudo reboot
 sudo apt install -y git htop joe jq \
   && wget -O nodejs.deb https://deb.nodesource.com/node_16.x/pool/main/n/nodejs/nodejs_16.17.1-deb-1nodesource1_amd64.deb \
   && sudo dpkg -i nodejs.deb \
-  && npm install pm2 -g
+  && sudo npm install pm2 -g
```

Comment:
Why do you think this is needed? Isn't it easier and better to install the npm packet pm2 for the current user only?

Reply:
for me it's not working without sudo. This is the output on 22.04.2 LTS

npm notice
npm notice New major version of npm available! 8.15.0 -> 9.6.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.3
npm notice Run npm install -g npm@9.6.3 to update!
npm notice
npm ERR! code EACCES
npm ERR! syscall rename
npm ERR! path /usr/lib/node_modules/pm2
npm ERR! dest /usr/lib/node_modules/.pm2-zkn4vkCL
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, rename '/usr/lib/node_modules/pm2' -> '/usr/lib/node_modules/.pm2-zkn4vkCL'
npm ERR!  [Error: EACCES: permission denied, rename '/usr/lib/node_modules/pm2' -> '/usr/lib/node_modules/.pm2-zkn4vkCL'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'rename',
npm ERR!   path: '/usr/lib/node_modules/pm2',
npm ERR!   dest: '/usr/lib/node_modules/.pm2-zkn4vkCL'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2023-03-31T12_28_49_939Z-debug-0.log